code/WorkInProgress/SpyGuyStuff.dm

```diff
@@ -1018,7 +1018,7 @@ proc/Create_Tommyname()
 
 	proc/check_conditions()
 		. = 0
-		if(BOUNDS_DIST(owner, target) > 0 || !target || !owner || !the_garrote || !the_garrote.wire_readied)
+		if(BOUNDS_DIST(owner, target) > 0 || !isturf(target.loc) || !target || !owner || !the_garrote || !the_garrote.wire_readied)
```

Comment:
```suggestion
		if(BOUNDS_DIST(owner, target) > 0 || !target || !isturf(target.loc) || !owner || !the_garrote || !the_garrote.wire_readied)
```
best to check if `target` exists before trying to access its `loc`

Reply:
Yeah probably, I didn't even notice that. Whole garrote code feels like a mess but that's beyond me

Reply:
Okay that made it commit your suggestion right?